src/GraphQL/DataLoader/DataLoaderDocumentListener.cs

```diff
@@ -34,6 +38,8 @@ public Task BeforeExecutionAwaitedAsync(object userContext, CancellationToken to
 
         public Task AfterExecutionAsync(object userContext, CancellationToken token)
         {
+            _accessor.Context = null;
```

Comment:
Should perhaps `DataLoaderContext` implement `IDisposable` and be disposed here, which would clear out any queue and loader references?

Reply:
I don’t think that would be necessary. There’s nothing that needs to be disposed of before garbage collection.

Reply:
This change probably isn’t necessary. I just wanted to make sure that the context would not be reused for another execution.